inventory/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+# Create your models here.
+from MembershipApp.models import BubblyMember
+
+class Item(models.Model):
+    member = models.ForeignKey(BubblyMember, on_delete=models.CASCADE)
+    title = models.CharField(max_length=127, blank=False)
+    description = models.TextField()
+    width = models.PositiveIntegerField(null=True, blank=True, help_text="in meters")
+    height = models.PositiveIntegerField(null=True, blank=True, help_text="in meters")
+    length = models.PositiveIntegerField(null=True, blank=True, help_text="in meters")
+    voltage = models.PositiveIntegerField(null=True, blank=True, help_text="If it is an electrical item")
+    wattage = models.PositiveIntegerField(null=True, blank=True, help_text="If it is an electrical item")
```

Comment:
no this is nto what we agreed on. 
We agreed to make a generic item `Item` and to create a more specific item `Electrical Item` that inherits from item and has extra attributes like wattage, voltage etc.

By doing this you're saying any item in the inventory will have this, which is irrelevant for an object like tent